jenkins/Jenkinsfile

```diff
@@ -8,22 +8,25 @@ node {
 
     stage('Testing') {
         def requirements = ['requirements.txt', 'test_requirements.txt']
-        def pytestArgs = '-v --tb=short tests'
+        def pytestArgs = '-v --tb=short -k "not weather_anomal" tests'
```

Comment:
This is going to change back to all tests soon (no -k option). Looks like that should be easy to update, yes?

Reply:
Yes, that won't be a problem.